--- a/app/main.py
+++ b/app/main.py
@@ -108,7 +108,7 @@
 @app.get('/-/readme', summary='* Readme')
 async def _readme():
     from markdown import markdown
-    content = load_file('../README.md').decode('utf-8')
+    content = load_file("/".join(__file__.split("/")[:-1]) + "/README.md").decode('utf-8')
     return HTMLr(markdown(text=content, extensions=['tables', 'fenced_code', 'md_in_html', 'nl2br', 'toc']))
